redesign/client-redesign/src/js/get-guid.js

```diff
@@ -0,0 +1,8 @@
+export default function () {
```

Comment:
Tohle neni uplne GUID (to ma jiny format) a Math.random() generuje pseudonahodna cisla, ale pro nase potreby to staci. Maximalne bych prejmenoval soubor na "get-random-identifier" nebo tak neco, ale to je drobnost. Pripadne to resit knihovnou https://www.npmjs.com/package/uuid, ale to je v teto fazi trosku overkill.

Reply:
Ano, v rámci a0fd6a9 jsem zmínil možná řešení, pro současné účely stačí. Soubor přejmenuji, díky. 👍